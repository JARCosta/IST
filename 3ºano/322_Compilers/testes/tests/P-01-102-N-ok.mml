[double] f;
begin
  f = [2];
  f[0] = 6.2E3;
  f[1] = 3.1E3;
  f[0]!!
  f[1]!!
  return 0;
end

