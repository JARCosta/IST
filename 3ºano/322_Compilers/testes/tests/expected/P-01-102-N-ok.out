6.2E3
3.1E3
